Realm/ObjectStore/object_schema.cpp

```diff
@@ -0,0 +1,67 @@
+////////////////////////////////////////////////////////////////////////////
+//
+// Copyright 2015 Realm Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////
+
+#include "object_schema.hpp"
+#include "object_store.hpp"
+
+#include <realm/group.hpp>
+#include <realm/table.hpp>
+
+using namespace realm;
+using namespace std;
+
+ObjectSchema::ObjectSchema(Group *group, const std::string &name) : name(name) {
```

Comment:
Our C++ code standard requires the { to be on the next line. (so does the Obj-C coding standard, although that's not being enforced).